extensions-core/protobuf-extensions/pom.xml

```diff
@@ -187,7 +187,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.2.1</version>
+        <version>3.4.1</version>
```

Comment:
I had to fix the same problem as part of https://github.com/apache/druid/pull/15014, any reason you did not upgrade to 3.5.0?

Reply:
also, why are we explicitly setting the plugin version here if it's already defined in the parent pom?

Reply:
no reason, i was trying to fix a issue in the unit tests where we were periodically getting: Error creating shaded jar: duplicate entry: META-INF/services/org.apache.druid.opentelemetry.shaded.io.grpc.NameResolverProvider if two builds ran two quickly one after another, and 3.4.1 was where the docs said this issue was fixed